Scripts/script-DBotPredictPhishingLabel.yml

```diff
@@ -0,0 +1,71 @@
+commonfields:
+  id: DBotPredictPhishingLabel
+  version: -1
+name: DBotPredictPhishingLabel
+script: |-
+  import json
+  args = demisto.args()
+  args['inputText'] = args.get('emailSubject','') + ' ' + args.get('emailBody','')
+  res = demisto.executeCommand('DBotPredictTextLabel', demisto.args())
+  res = res[-1]
+  old_context_key_prefix = 'DBotPredictTextLabel'
+  new_context_key_prefix = 'DBotPredictPhishingLabel'
+  ec = res.get('EntryContext', {}) or {}
```

Comment:
`or {}` is redundant here

Reply:
it's not believe me :), sometimes it's return explicit None cuz it's JSON parse